syncany-lib/src/main/java/org/syncany/operations/ActionFileHandler.java

```diff
@@ -94,25 +95,27 @@ private void deleteActionFile(ActionRemoteFile actionFile) throws StorageExcepti
 		transferManager.delete(actionFile);
 	}
 
-	private void uploadActionFile(ActionRemoteFile actionFile) throws Exception {
+	private void uploadActionFile(ActionRemoteFile actionFile) throws StorageException, IOException {
 		logger.log(Level.INFO, "Uploading action file: " + actionFile);
 
-		File tempActionFile = File.createTempFile("syncany-action-", ".tmp");
+		File tempActionFile = null;
+		tempActionFile = File.createTempFile("syncany-action-", ".tmp");
```

Comment:
Oops, I thought I'd already put that back (I'm not sure where this came from. Fixed in new commit).